libs/clients/adr-and-machine-license/src/lib/adr-and-machine-license.config.ts

```diff
@@ -20,7 +20,7 @@ export const AdrAndMachineLicenseClientConfig = defineConfig<
       'IS-DEV/GOV/10013/Vinnueftirlitid-Protected/rettindi-token-v1',
     ),
     fetch: {
-      timeout: 30000,
+      timeout: env.optionalJSON('ADR_LICENSE_FETCH_TIMEOUT') ?? 10000,
```

Comment:
Is there a place where it would make sens to store something like
`const DEFAULT_LICENSE_FETCH_TIMEOUT = 10000`?

Reply:
just manually defined secret, doesn't need to be a parameter really

Reply:
yes, we'd do this solely in code, just where would we put it

Reply:
defer this till later, unclear where it should be stored. This would be a good use case for the general license client